src/partials/menu.html.eco

```diff
@@ -42,6 +42,11 @@
                         <span itemprop="name">Resources</span>
                     </a>
                 </li>
+                <li class="nav-item <% if @isActiveMenu('assets'): %>nav-active<% end %>">
+                    <a href="<%= @site.url %>/assets/" class="nav-link" title="Assets" itemprop="url">
+                        <span itemprop="name">Assets</span>
+                    </a>
+                </li>
```

Comment:
A link should be included under the "Community" section on the footer instead of main menu